examples/player/simid_player.html

```diff
@@ -147,22 +153,38 @@
       <tr>
         <td>Ad type</td>
         <td>
-          <input type="radio" id="linear_ad" name="ad_type" checked> <label for="linear_ad" class="ad_type_label">Linear
+          <input type="radio" id="linear_ad" name="ad_type" onclick="toggleInputs()"> <label for="linear_ad" class="ad_type_label">Linear
```

Comment:
before we were specifying that this was checked. Does this still start as checked by default?

Reply:
^ why get rid of where it defualts

Reply:
No, it doesn't start as checked. I could make linear checked on default though